### PR TITLE
Add parameter for minimumInputLength

### PR DESCRIPTION
**Pull request by philkingston**

### CODE REVIEW

File: Form/Type/AjaxEntityType.php [hunk 1]
```diff
@@ -115,6 +117,7 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
                 'repo_method'   => null,
                 'property'      => null,
                 'multiple'      => false,
+        		'minimum_input_length' => 3,
```
Comment (kbond) [inline Form/Type/AjaxEntityType.php]: Can you match the indentation here?

Comment (philkingston) [inline Form/Type/AjaxEntityType.php]: Done. Do you need a new pull request?

Comment (kbond) [inline Form/Type/AjaxEntityType.php]: Nope, all good!

**Comment by kbond** [comment]
Thanks @philkingston!

**Pull request merged**